app/src/main/java/org/sopt/dosopttemplate/presentation/mainhome/MainHomeActivity.kt

```diff
@@ -43,6 +46,20 @@ class MainHomeActivity : AppCompatActivity() {
         clickBottomNavigation()
     }
 
+    private fun clickBottomNavigation() {
+        var userId= intent.getIntExtra("id", -1)
```

Comment:
중요하지 않지만222 포맷팅하면 좋을 것 같아요 ㅎㅎ !! 
```suggestion
        var userId = intent.getIntExtra("id", -1)
```
공백 추가하기 !!!!!!!

Reply:
했슴당~!!!